[2024-10-15T17:42:43.495+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-15T17:42:43.516+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: main_v10.run_archive_function manual__2024-10-15T17:42:39.484606+00:00 [queued]>
[2024-10-15T17:42:43.526+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: main_v10.run_archive_function manual__2024-10-15T17:42:39.484606+00:00 [queued]>
[2024-10-15T17:42:43.526+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-10-15T17:42:43.651+0000] {taskinstance.py:2888} INFO - Executing <Task(PostgresOperator): run_archive_function> on 2024-10-15 17:42:39.484606+00:00
[2024-10-15T17:42:43.657+0000] {standard_task_runner.py:72} INFO - Started process 477 to run task
[2024-10-15T17:42:43.664+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'main_v10', 'run_archive_function', 'manual__2024-10-15T17:42:39.484606+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmp_v5k_kyy']
[2024-10-15T17:42:43.669+0000] {standard_task_runner.py:105} INFO - Job 28: Subtask run_archive_function
[2024-10-15T17:42:43.718+0000] {task_command.py:467} INFO - Running <TaskInstance: main_v10.run_archive_function manual__2024-10-15T17:42:39.484606+00:00 [running]> on host d4bd6f97aac1
[2024-10-15T17:42:43.803+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='main_v10' AIRFLOW_CTX_TASK_ID='run_archive_function' AIRFLOW_CTX_EXECUTION_DATE='2024-10-15T17:42:39.484606+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-15T17:42:39.484606+00:00'
[2024-10-15T17:42:43.803+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-15T17:42:43.815+0000] {sql.py:274} INFO - Executing: SELECT archive_old_data();
[2024-10-15T17:42:43.825+0000] {base.py:84} INFO - Retrieving connection 'postgresid'
[2024-10-15T17:42:43.836+0000] {base.py:84} INFO - Retrieving connection 'postgresid'
[2024-10-15T17:42:43.841+0000] {sql.py:418} INFO - Running statement: SELECT archive_old_data();, parameters: None
[2024-10-15T17:42:43.843+0000] {sql.py:427} INFO - Rows affected: 1
[2024-10-15T17:42:43.865+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-15T17:42:43.866+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=main_v10, task_id=run_archive_function, run_id=manual__2024-10-15T17:42:39.484606+00:00, execution_date=20241015T174239, start_date=20241015T174243, end_date=20241015T174243
[2024-10-15T17:42:43.914+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-15T17:42:43.947+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-15T17:42:43.949+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
